packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx

```diff
@@ -0,0 +1,182 @@
+    [pgnUrl, setPgnUrl, 'Tournament Url'],
+    [arbiterPin, setArbiterPin, 'Arbiter Pin'],
+    [arbiterEmailSecret, setArbiterEmailSecret, 'Arbiter Email Secret'],
+    [divisionId, setDivisionId, 'Division Id'],
+    [arbiterId, setArbiterId, 'Arbiter Id'],
+    [arbiterName, setArbiterName, 'Arbiter Name'],
+    [eventName, setEventName, 'Event Name'],
+  ];
+
+  const validateArbiterInfo = (arbiterInfo: ArbiterInfo): boolean => {
+    if (arbiterInfo.arbiterToken == '') {
+      showError('Arbiter Email token cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.divisionId == '') {
+      showError('Division Id cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.eventName == '') {
+      showError('Event Name cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.userName == '') {
+      showError('Arbiter Name cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.userId == '') {
+      showError('Arbiter Id cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.pin == '') {
+      showError('Arbiter Pin cannot be blank');
+      return false;
+    }
+    if (!validUrl(arbiterInfo.broadcastUrl)) {
+      showError('Invalid Pgn Url');
+      return false;
+    }
+    return true;
+  };
+
+  const confirmArbiterInfo = async (): Promise<void> => {
+    const newArbiterInfo: ArbiterInfo = {
+      pin: arbiterPin,
+      broadcastUrl: pgnUrl,
+      arbiterToken: arbiterEmailSecret,
+      divisionId,
+      userId: arbiterId,
+      userName: arbiterName,
+      eventName,
+    };
+    dismiss();
+
+    if (validateArbiterInfo(newArbiterInfo)) {
+      await setArbiterInfo(newArbiterInfo);
+    }
+  };
+  return (
+    <>
+      <Sheet title="Edit Event" dismiss={dismiss} visible={display}>
+        <View style={styles.editEventHeader}>
+          <TextIconButton
+            Icon={inScanQRMode ? ICON_TEXT : ICON_QR}
+            text={inScanQRMode ? 'Text input' : 'QR input'}
+            buttonHeight={60}
+            style={styles.switchInputModeButton}
+            buttonTextStyle={styles.switchInputModeButtonText}
+            onPress={() => setInScanQRMode(!inScanQRMode)}
+          />
+          <PrimaryText
+            size={26}
+            weight={FontWeight.Medium}
+            style={styles.instructions}>
+            Go to{' '}
+            <Link
+              style={styles.instructionsLink}
+              label="tornelo.com"
+              link={torneloUrl}
+            />{' '}
+            and either generate a QR code or manually edit the fields.
+          </PrimaryText>
+        </View>
+        {/* QR Code Entry */}
+        {inScanQRMode && (
+          <View style={styles.editEventSheetSection}>
+            {/* Placeholder - QR Code scanning camera section will go here call confirmArbiterInfo() once the qr code has scanned and everything will be saved!*/}
+            <View
+              style={{
+                backgroundColor: colours.grey10,
```

Comment:
I know it's a placeholder but in our contributing guide it says we should have this stuff in the style file before we merge into main.